pkg/NAMESPACE

```diff
@@ -97,3 +103,5 @@ export("jsonFile",
        "uncacheTable")
 
 export("sparkRSQL.init")
+export("print.structType",
```

Comment:
Those are also `namespace` exports though. They also have things like
```R
export(knit_print.html)
export(knit_print.shiny.appobj)
export(knit_print.shiny.render.function)
export(knit_print.shiny.tag)
export(knit_print.shiny.tag.list)
```
in their namespace for the same reason. Additionally, we have `print.jobj` in our namespace in order for the method to trigger correctly. I don't see where the issue is in this case.